pkg/standalone/uninstall.go

```diff
@@ -10,6 +11,11 @@ import (
 
 // Uninstall deletes all installed containers
 func Uninstall(uninstallAll bool, dockerNetwork string) error {
+	dockerInstalled := utils.IsDockerInstalled()
+	if !dockerInstalled {
+		return errors.New("could not connect to Docker. Docker may not be installed or running")
```

Comment:
nit: end with punctuation.

Reply:
nit: While we are here can you please change "could not..." -> "Could not..."

Reply:
The linter doesn't like an errors.New that starts with a capital letter.

Reply:
That's right just checked the lint error. Neither should it end with punctuation. Sorry about the trouble @ksashikumar

Reply:
My bad! 🙁 I didn't run lint before pushing, I'll update in the next commit! 👍